graph/proto/substreams.proto

```diff
@@ -0,0 +1,267 @@
+syntax = "proto3";
```

Comment:
I had added this to chain/substreams/proto/substreams.proto, in this case I guess we can remove that one and the extra deps that were needed to build it.